docs/getting-started/configure-app-with-plugins.md

```diff
@@ -6,7 +6,66 @@ description: Documentation on How Configuring App with plugins
 
 ## Adding existing plugins to your app
 
-Coming soon!
+The following steps assumes you have created a new Backstage app and want to add
+an existing plugin to it. We are using the
+[CircleCI](https://github.com/backstage/backstage/blob/master/plugins/circleci/README.md)
+in this example.
+
+0. Add to the repo:
+
+```bash
+yarn add @backstage/plugin-circleci
+```
+
+1. Add plugin API to your Backstage instance:
+
+```js
+// packages/app/src/api.ts
```

Comment:
hm this is `apis.ts` and does not look similar to the actual code in there - double check with the latest app to see what the structure is like now